src/sphinxcontrib/bibtex/__init__.py

```diff
@@ -50,6 +55,19 @@ def setup(app: Sphinx) -> Dict[str, Any]:
     app.add_directive("footbibliography", FootBibliographyDirective)
     app.add_role("footcite", FootCiteRole())
 
+    # Catch bug in newer docutils
+    if (0, 18, 0, 'final', 0, True) <= docutils.__version_info__ < (0, 20, 0, 'final', 0, True):
```

Comment:
``(0, 18) <= docutils.__version_info__ < (0, 20)`` should do the trick

Reply:
Oh, I had forgotten about this feature of tuple comparisons. It will differ for the lower upper bounds (`(0, 18) <= docutils.__version_info__` includes all pre-releases of 0.18, whilst `docutils.__version_info__ < (0, 20)` includes no pre-releases of 0.20) but I think that's what we want here.

Reply:
Yeah, I think so. I believe this is consistent with what ``install_requires`` does (though pip will not include pre-releases unless explicitly instructed), i.e. this comparison should be identical to ``docutils>=0.18,<0.20``.